docs/secure-coding-guidelines.md

```diff
@@ -147,12 +147,19 @@ The guidelines in this policy were gathered primarily from the [OWASP Top 10](ht
 
 #### LavaMoat (JavaScript projects only)
 
-- LavaMoat `allow-scripts` should be enabled on all projects
-  - This project relies upon install scripts being disabled in your package manager. This can be verified by adding the dependency `@lavamoat/preinstall-always-fail`, which will cause installation to fail if scripts are enabled.
-  - `allow-scripts` acts as an allowlist for install scripts. Use the `allow-scripts` binary after installing dependencies to run install scripts on the allowlist.
-    - On Yarn v3 projects, use the `yarn-plugin-allow-scripts` plugin to run allowed scripts automatically during install
-    - On other projects, use an npm script called `setup` that will call `install` and `allow-scripts` in sequence
-  - If you're unsure whether an install script is needed, leave it disabled
+- LavaMoat `allow-scripts` should be enabled on all projects. This project uses an allowlist to run trusted install scripts, ensuring new or untrusted scripts are never run unexpectedly.
+
+  This project relies upon install scripts being disabled in your package manager. Install scripts should be run explicitly via `allow-scripts` instead, which uses the allowlist configured in `package.json`.
+
+  See [the `allow-scripts` README](https://github.com/LavaMoat/LavaMoat/tree/main/packages/allow-scripts) for setup and configuration instructions.
+
+  After setup and configuration, ensure that `allow-scripts` is run every time you install dependencies as part of your development workflow. There are two approaches to this:
+
+  - On Yarn v3 projects, use [the `yarn-plugin-allow-scripts` plugin](https://github.com/LavaMoat/LavaMoat/tree/main/packages/yarn-plugin-allow-scripts) to run allowed scripts automatically during install
+  - On other projects, use an npm script called `setup` that will call `install` and `allow-scripts` in sequence
+
+  Each time a dependency is added or updated, run `yarn allow-scripts auto` to detect new install scripts. They should be added to the configuration automatically, disabled by default. Review each one, only enabling those that are necessary. If you notice anything suspicious about the script or the package (e.g. if the maintainer has recently changed, or if the script appears to be obfuscated), consult with the security team before enabling it.
```

Comment:
- look up `socket.dev/npm/package/PACKAGENAME` to get hints - they can detect malicious scripts

Reply:
Good idea! I have pushed a new commit to expand these review guidelines

Reply:
There's also npmjs.com/package/can-i-ignore-scripts but I haven't put much work into it recently.